startPage.js

```diff
@@ -0,0 +1,48 @@
+let gameStory = " Sometimes it can be hard to balance our life. This game will give you the possibility of      playing with different aspects that we considered important in order to have a good           balance. There are 4 sections and they will add up depending on the items you grab.        Once you achieve 10 items of each section the game will be won but... be  carefull since   there are robots that will kill you once you collide against them and there is only one         question to be answered here... Are you ready to start gaming with your life? "
+
+
+
+// creat text class for writing on canvas
+class text {
+    constructor(string, x, y, lineHeight) {
+        this.string = string;
+        this.x = x;
+        this.y = y;
+        this.lineHeight = lineHeight;
+        this.lineStartX = x;
+    }
+    // method to write it on the canvas letter by latter
+    update(speed) {
+        // declare an i counter to get the index of the letters from the string
+        let i = -30;
+        // declare an j counter to start writing on the canvas
+        let j = 0;
+
+        let typeText = setInterval(() => {
```

Comment:
here I would name the function instead of anonymous function, so that the `speed` doesn't hang in the middle of nowhere in the end ;)